### ISSUE TITLE
Dependency name 'stdlib' found, but expected 'dp-stdlib' instead

### ISSUE THREAD

**Issue by ghost** [zoziha/dp-stdlib#3]
@zoziha thanks for lightweight stdlib. I was following the instructions in README.md and ran into issue

```
fpm build
…
From https://github.com/zoziha/dp-stdlib
 * branch            HEAD       -> FETCH_HEAD
<ERROR>*cmd_build*:model error:Dependency name 'stdlib' found, but expected 'dp-stdlib' instead
STOP 1
```

I think this will work:
- change the library name in fpm.toml, or
- change the ~~library~~ dependency name in README.md

**Comment by zoziha** [comment]
Yes, this is `dp-stdlib` that I put together temporarily yesterday, and it has this problem. 
I've just tried to solve the existing problems, and `stdlib_random` is probably unsafe (incorrect) right now, I'm going to keep an eye on it.
